modules/node_groups/README.md

```diff
@@ -23,6 +23,8 @@ The role ARN specified in `var.default_iam_role_arn` will be used by default. In
 | capacity\_type | Type of instance capacity to provision. Options are `ON_DEMAND` and `SPOT` | string | Provider default behavior |
 | create_launch_template | Create and use a default launch template | bool |  `false` |
 | desired\_capacity | Desired number of workers | number | `var.workers_group_defaults[asg_desired_capacity]` |
+| disk\_encrypted | Whether the root disk will be encrypyted. Require `create_launch_template` to be `true` and require `disk_kms_key_id` to be set | bool | false |
+| disk\_kms\_key\_id | KMS Key used to encrypt the root disk. Require both `create_launch_template` and `disk_encrypted` to be `true` | string | "" |
```

Comment:
```suggestion
| disk\_encrypted | Whether the root disk will be encrypyted. Requires `create_launch_template` to be `true` and require `disk_kms_key_id` to be set | bool | false |
| disk\_kms\_key\_id | KMS Key used to encrypt the root disk. Requires both `create_launch_template` and `disk_encrypted` to be `true` | string | "" |
```

Reply:
I can certainly change this. I elected to use the singular as it was the existing style in the other README entries for the sake of consistency, though I agree it's a little stilted.

Reply:
It's more than just stilted, it's grammatically incorrect.

Just noticed there's a second "require" on line 26 that also has incorrect number. It should probably be removed.

Reply:
Sorry for the delay. I updated the language in the README